### PR TITLE
doc/foundation: add Samsung Electronics

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: doc/foundation.rst [hunk 1]
```diff
@@ -34,6 +34,7 @@ Premier
 * `Intel <http://www.intel.com/>`_
 * `OVH <https://www.ovh.com/>`_
 * `Red Hat <https://www.redhat.com/>`_
+* `Samsung <https://samsung.com/>`_
```
Comment (myoungwon) [inline doc/foundation.rst]: Samsung Electronics

Comment (tchaikov) [reply doc/foundation.rst]: @myoungwon fixed and repushed

**Comment by myoungwon** [comment]
Reviewed-by: Myougnwon Oh <myoungwon.oh@samsung.com>

**Comment by liewegas** [comment]
retest this please

**Pull request merged**